### PR TITLE
Update scroll-node.md

### PR DESCRIPTION
**Pull request by ArEnSc**
Updating with corresponding 2.0 release.
Just a small pull request before I add more doc and some examples

### CODE REVIEW

File: docs/_docs/scroll-node.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ Say you have a giant image, but you only want to take up 200x200 pts on the scre
 UIImage *scrollNodeImage = [UIImage imageNamed:@"image"];
 ASScrollNode *scrollNode = [[ASScrollNode alloc] init];
 
-scrollNode.preferredFrameSize = CGSizeMake(200.0, 200.0);
+scrollNode.style.prefferedSize = CGSizeMake(200.0, 200.0);
```
Comment (hannahmbanana) [inline docs/_docs/scroll-node.md]: Could you fix the spelling mistake - should be `.preferredSize`

File: docs/_docs/scroll-node.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ scrollNodeView.contentSize = scrollNodeImage.size;
 let scrollNodeImage = UIImage(named: "image")
 let scrollNode = ASScrollNode()
 
-scrollNode.preferredFrameSize = CGSize(width: 200.0, height: 200.0)
+scrollNode.style.prefferedSize = CGSize(width: 200.0, height: 200.0)
```
Comment (hannahmbanana) [inline docs/_docs/scroll-node.md]: same here

**Comment by hannahmbanana** [comment]
thanks @ArEnSc for taking the time to update the docs!!!

**Review by ArEnSc — COMMENTED** [review]
Fixed spelling sorry slightly dyslexic =)

**Pull request merged**